src/ParseFile.js

```diff
@@ -134,10 +134,16 @@ export default class ParseFile {
    * Gets the url of the file. It is only available after you save the file or
    * after you get the file from a Parse.Object.
    * @method url
+   * @param {Object} options An object to specify url options
    * @return {String}
    */
-  url(): ?string {
-    return this._url;
+  url(options?: { secure?: boolean }): ?string {
```

Comment:
How about changing `secure` to `forceSecure`? If the server ever decided to default to https, the API would still be correct.